examples/js/calendar-widget/index.html

```diff
@@ -1,12 +1,8 @@
 <html>
   <head>
-    <!--
-      Do not use @8 in production, use a complete version like x.x.x, see website for latest version:
-      https://www.algolia.com/doc/guides/building-search-ui/installation/js/#load-the-styles
-    -->
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/instantsearch.css@8/themes/algolia.min.css"
+      href="/packages/instantsearch.css/themes/algolia-min.css"
```

Comment:
this won't work on codesandbox, which is why jsdelivr is used in the first place. putting the import in js also doesn't really work because this example works without bundling too. Not sure what the right solution is though

Reply:
I changed it with module imports, it didn't need to be imported like that anyway.